src/diffusers/pipelines/blip_diffusion/modeling_blip2.py

```diff
@@ -0,0 +1,653 @@
+# coding=utf-8
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" PyTorch BLIP-2 model."""
```

Comment:
Maybe make it clear that:

* This exists because there's currently not support for getting multimodal embeddings. 
* There's a plan on getting that merged in `transformers` and when it's in this module will go away and instead, we're directly leverage it from `transformers`.

Reply:
Done